test/integration/testlib/glob.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+from .notification import NotificationServer
+
+notifications = NotificationServer()
```

Comment:
Hm, why not have notifications.py provide a global `notifications` variable? Also, glob.py is a weird name, it conflicts with the [`glob` module](https://docs.python.org/3/library/glob.html).

Reply:
It was a solution for circular imports that's not needed anymore and was simply missed. Thanks for noticing, I'll merge it.